### PR TITLE
UI fixes for failing model serving tests

### PR DESCRIPTION
**Pull request by ConorOM1**
Some updates to the UI requires dashboard fixes for the following model serving tests:
ODS-2626
ODS-2054

### CODE REVIEW

File: ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHModelServing.resource [hunk 1]
```diff
@@ -437,36 +436,14 @@ Deploy Model Via CLI
 Deploy Kserve Model Via UI  #robocop: disable
     [Documentation]  Deploys a model using the kserve/caikit runtime using the Data Science Projects UI
     [Arguments]    ${model_name}    ${serving_runtime}    ${data_connection}    ${path}
-    ...    ${model_framework}=caikit    ${replicas}=1    ${size}=Small    ${no_gpus}=${0}
+    ...    ${model_framework}    ${replicas}=1    ${size}=Small    ${no_gpus}=${0}
```
Comment (FedeAlonso) [inline ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHModelServing.resource:439]:
```suggestion
    ...    ${model_framework}=caikit    ${replicas}=1    ${size}=Small    ${no_gpus}=${0}
```

Comment (ConorOM1) [reply ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHModelServing.resource]: reverted

**Review by CFSNM — APPROVED** [review]

**Review by jiridanek — APPROVED** [review]

**Comment by ConorOM1** [comment]
thank you for reviewing @CFSNM & @jiridanek , can you merge please ? I don't have authorization at the moment

**Comment by CFSNM** [comment]
merged @ConorOM1

**Pull request merged**